### ISSUE TITLE
Use DevTools to view Elements to expand the arrow icon to display incomplete

### ISSUE THREAD

**Issue by A81N9** [Alex313031/Thorium-Win#307]
Use DevTools to view Elements to expand the arrow icon to display incomplete

![image](https://github.com/user-attachments/assets/aada4bd6-0a13-48dc-936a-fc204ec688ea)

**Comment by gz83** [comment]
Can you please tell me your current screen resolution, screen zoom size, and the flags you are currently using?

In addition, your devtool currently seems to be using Chinese as the display language.

**Comment by A81N9** [comment]
> Can you please tell me your current screen resolution, screen zoom size, and the flags you are currently using?
> 
> In addition, your devtool currently seems to be using Chinese as the display language.

Default flags
Screen resolution: 1920*1080
Screen zoom size: 100%

Yes, my DevTools uses Chinese, but after my test, switching DevTools with English still exists in this problem.

**Comment by gz83** [comment]
Do you have this problem on Chrome as well? Also, have you tried disabling all extensions?

**Comment by A81N9** [comment]
Yes, I tried the same problem in incognito mode, but I didn't find it in Chrome and Edge

**Comment by A81N9** [comment]
@gz83 I discovered the problem today, which is caused by the automatic wrap of elements in the devtools.

<img width="579" height="886" alt="Image" src="https://github.com/user-attachments/assets/3dd7b922-39aa-4004-805d-afff07363811" />

**Comment by gz83** [comment]
Please try the latest version 138. If you still have problems, please let us know.

**Comment by A81N9** [comment]
<img width="375" height="117" alt="Image" src="https://github.com/user-attachments/assets/4716c380-12e5-4369-bd05-9b81038c4fe4" />

@gz83  I just tested this on version 138.0.7204.303 and the problem still exists.

**Comment by gz83** [comment]
Could you test Chrome/Chromium to see if you encounter the same problem?

**Comment by A81N9** [comment]
<img width="568" height="373" alt="Image" src="https://github.com/user-attachments/assets/3e754e36-d4a3-4b5a-8d88-9416fc8739e8" />

I also have this problem when testing with Chrome 145.

**Comment by gz83** [comment]
Could you please submit an issue report regarding the problem you encountered?

https://issues.chromium.org/issues/new?component=1457255&template=1923529

**Comment by A81N9** [comment]
https://issues.chromium.org/issues/487131212

**Comment by gz83** [comment]
Please upload the correct screenshots again to the chromium issue tracker.

**Comment by gz83** [comment]
I am unable to reproduce the issue you are experiencing locally, perhaps the problem only occurs on Windows devices? But in any case, I want to see if there is any response in the upstream issue report first.

**Comment by A81N9** [comment]
@gz83 After my testing, this issue is related to browser custom font settings, like some fonts cause line height to be truncated.
I changed the font configuration to the following to solve this problem.

<img width="578" height="840" alt="Image" src="https://github.com/user-attachments/assets/810fd4f3-015a-4bf8-8d21-cb959754af5c" />

**Comment by gz83** [comment]
Please keep this issue open for now and explain what font you used previously. I think this is still an issue.

**Comment by A81N9** [comment]
Before, except for math fonts, all other fonts I used were `微软雅黑` fonts, instead of `Microsoft YaHei UI` font.

**Comment by gz83** [comment]
This is not a problem unique to Thorium; let's leave it to the upstream to solve.
